.github/workflows/go.yml

```diff
@@ -231,8 +231,8 @@ jobs:
       job-name: autopilot-test
       k0s-reference-version: ${{ matrix.version }}
 
-  smoketest-arm:
-    name: Smoke test on armv7/arm64
+  build-arm:
+    name: build on armv7/arm64
```

Comment:
Every other name starts with a capital case. Other than that LGTM and I don't want to block a PR for something this small. Either merge it as is or modify it and ping me on slack so that I approve quickly.